apps/models.py

```diff
@@ -164,6 +164,12 @@ class AppInstance(models.Model):
         on_delete=models.CASCADE,
         related_name="appinstance",
     )
+    flavor = models.ForeignKey(
+        "projects.Flavor",
+        on_delete=models.CASCADE,
+        related_name="appinstance",
+        null=True,
+    )
```

Comment:
This adds flavor to `Appinstance`. I added `on_delete` condition so if we remove a flavor from a project, it will automatically remove the appinstances that have that flavor.